docs/dasharo-menu-docs/dasharo-system-features.md

```diff
@@ -0,0 +1,110 @@
+# Dasharo System Features
+
+When entering the `Dasharo System Features` menu, one may see the following
+submenus to appear:
+
+- [Dasharo System Features](#dasharo-system-features)
```

Comment:
@miczyg1 Do you feel we need this ToC created manually here?
The options are also visible in the screenshot, so no need to provide them again here, IMHO,

We can apply following diff to mkdocs.yml settings:

```
diff --git a/mkdocs.yml b/mkdocs.yml
index 96b17b7bc883..a868088f6e1b 100644
--- a/mkdocs.yml
+++ b/mkdocs.yml
@@ -26,7 +26,8 @@ markdown_extensions:
   - attr_list
   - meta
   - toc:
-      toc_depth: 2
+      permalink: true
+      toc_depth: 4
   - pymdownx.tasklist:
       custom_checkbox: true
   - pymdownx.superfences
   ```
   
   This will give us for free, ToC created automatically in docs, which makes it easy to switch between sections.

Reply:
See the ToC on the  right.
 
 The existing ToC in text seems awkward to use, IMHO.
 
![2023-01-16T19:40:25,985385574+01:00](https://user-images.githubusercontent.com/19353650/212746801-2c2f99b9-fc05-42b7-a6a4-8e349076ba44.png)

Reply:
Ok, maybe this file is not the best example, as you had 2nd-level headers here already, so they were already visible in the ToC on the right.

Reply:
Thanks for the pointer to toc depth. I added it manually because of the depth level did not reach the headers I wanted to be shown on the right.

Reply:
It doesn't work as I would like it. There are no indentations on the lower level headers e.g. in the boot maintenance manager I have

```
* [Boot Maintnenace Manager](#boot-maintenance-manager)
    - [Boot Options](#boot-options)
        + [Add Boot Option](#add-boot-option)
        + [Delete Boot Option](#delete-boot-option)
        + [Change Boot Order](#change-boot-order)
```

But the deepest 3 are on an equal level of Boot Options. I guess we can't do anything about it, but still the TOC on right should be sufficient.

Reply:
TOCs removed.